libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/SpeechAnnouncement.java

```diff
@@ -37,6 +37,17 @@
    */
   public abstract String announcement();
 
+  /**
+   * Convert the current {@link SpeechAnnouncement} to its builder holding the currently assigned
+   * values. This allows you to modify a single property and then rebuild the object resulting in
+   * an updated and modified {@link SpeechAnnouncement}.
```

Comment:
This wording sounds like it is mutable, maybe change the language to make it clear that it's creating a new object?

Reply:
Good call - updated